infra/src/main/kotlin/fi/fta/geoviite/infra/linking/Linking.kt

```diff
@@ -40,16 +40,17 @@ data class EmptyAlignmentLinkingParameters<T>(
 
 data class LocationTrackSaveRequest(
     val name: AlignmentName,
-    val description: FreeText,
+    val descriptionBase: FreeText,
+    val descriptionSuffix: DescriptionSuffixType,
     val type: LocationTrackType,
     val state: LayoutState,
     val trackNumberId: IntId<TrackLayoutTrackNumber>,
     val duplicateOf: IntId<LocationTrack>?,
     val topologicalConnectivity: TopologicalConnectivityType,
 ) {
     init {
-        require(description.length in 4..256) {
-            "LocationTrack description length ${description.length} not in range 4-256"
+        require(descriptionBase.length in 4..256) {
+            "LocationTrack description length ${descriptionBase.length} not in range 4-256"
```

Comment:
Koko kuvauksen tutkiminen menisi tässä vaikeaksi -> tutkitaan pelkkä perusosa

Reply:
Hyvä näin, pointtihan näissä on validoida käyttäjän inputteja ja tuo automaattiosa ei ole sellainen :+1: